test/e2e/tests/simulation-details/mock-request-error-insuffient-gas.ts

```diff
@@ -12,7 +12,6 @@ export const INSUFFICIENT_GAS_TRANSACTION_MOCK = {
 
 export const INSUFFICIENT_GAS_REQUEST_MOCK: MockRequestResponse = {
   request: {
-    id: '0',
```

Comment:
this id is removed as was preventing for a request to be mocked:

![Screenshot from 2025-05-30 12-09-22](https://github.com/user-attachments/assets/5d8f00a1-ba17-4551-a4b1-4a315c144a29)

Reply:
:information_source:  the 2 reqs are expected, confirmed by the Confirmations team.
After digging deeper it seems that in this case, given the error insufficient funds for gas, we should only expect one request, but 2 are happening, with the exact same params, except for the `blockOverrides` param being passed two. So we still need to mock 2 requests, otherwise the last one goes to the catch all mock and the test fails.